src/test/java/careconnect/model/person/UniquePersonListTest.java

```diff
@@ -5,9 +5,9 @@
 import static careconnect.testutil.Assert.assertThrows;
 import static careconnect.testutil.TypicalPersons.ALICE;
 import static careconnect.testutil.TypicalPersons.BOB;
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertFalse;
-import static org.junit.jupiter.api.Assertions.assertTrue;
+//CHECKSTYLE.OFF: AvoidStarImport
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
i think we should list down all imports separately as according to the style guide. If this is because intellij automatically does combine the imports, try - Settings -> Editor -> Code Style -> Java -> Imports and increase "Class count to use *" :)